code/modules/clothing/clothing.dm

```diff
@@ -492,6 +492,8 @@
 	if(!no_slip)
 		return
 	var/mob/living/carbon/human/H = user
+	if(!user)
+    	return
```

Comment:
Using spaces as indents, vcs and git is CRINGE and wont let suggested changes fix it